### PR TITLE
bpo-40275: import locale lazily in gettext

### PR DESCRIPTION
**Pull request by shihai1991**
https://bugs.python.org/issue40275

### CODE REVIEW

File: Lib/gettext.py [hunk 1]
```diff
@@ -296,6 +297,7 @@ def ngettext(self, msgid1, msgid2, n):
             return msgid2
 
     def lngettext(self, msgid1, msgid2, n):
+        import locale
```
Comment (vstinner) [inline Lib/gettext.py]: Can you move the locale import after the warn() call? Here in the following function.

Comment (shihai1991) [reply Lib/gettext.py]: ok, done.

**Review by vstinner — APPROVED** [review]
LGTM.

@JulienPalard @serhiy-storchaka: Would you mind to review this change?

**Comment by vstinner** [comment]
Thanks @shihai1991! I merged your PR.

**Comment by shihai1991** [comment]
> Thanks @shihai1991! I merged your PR.

Thanks, victor.

**Pull request merged**